style.css

```diff
@@ -100,3 +100,315 @@ header {
+
+.client {
+  font-style: normal;
+  font-weight: 400;
+  text-transform: uppercase;
+  color: #344563;
+}
+
+span {
+  margin-right: 12px;
+  text-decoration: none;
+  font-size: 0.8em;
+  font-family: 'Poppins', sans-serif;
+}
+
+.fa-circle {
+  font-size: 60px;
+  color: #c1c7d0;
+  text-decoration: none;
+}
+
+.card-primary-text {
+  width: 295px;
+  height: 76px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+}
+
+.tags {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+}
+
+.tag {
+  align-items: center;
+  margin: auto auto;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+}
+
+button {
+  display: flex;
+  justify-content: center;
+  margin: 16px 0 26px 0;
+  padding: 12px 12px;
+  color: #6070ff;
+  box-sizing: border-box;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: white;
+  text-transform: uppercase;
+  font-family: 'Poppins', sans-serif;
+}
+
+.role .year {
+  font-style: normal;
+  font-weight: 400;
+  color: #7a869a;
+  font-family: 'Poppins', sans-serif;
+}
+
+.works-2 {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 5px;
+  background-color: white;
+  width: 85%;
+  height: auto;
+  margin: 114px auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.works-3 {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 5px;
+  background-color: white;
+  width: 85%;
+  height: auto;
+  margin: 114px auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.works-4 {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 5px;
+  background-color: white;
+  width: 85%;
+  height: auto;
+  margin: 114px auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-works-title-2 {
+  font-style: normal;
+  width: 200px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  font-family: 'Poppins', serif;
+}
+
+.about-my-self-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 44px;
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  margin-left: 24px;
+  padding-top: 114px;
+}
+
+.paragraph {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 20px;
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  width: 330px;
+  margin-left: 24px;
+}
+
+.action {
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 24px;
+}
+
+.about-my-self-button {
+  margin: 16px 0 26px 0;
+  padding: 12px 12px;
+  color: #6070ff;
+  box-sizing: border-box;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: white;
+  text-transform: uppercase;
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
- [ ] It's awesome 👍 that you have implemented a `button` tag for `Get My Resume`. Kindly use hover and click event as per the project requirement.